lib/improver/tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -135,11 +135,28 @@ def set_up_wxcubes():
             forecast_thresholds=np.array([1000.0, 5000.0])))
     visibility.attributes['relative_to_threshold'] = 'below'
 
+    data_lightning = np.array([0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many positional arguments for method call](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4535748)